tests/microos/workloads/x11-container/x11_helm_server.pm

```diff
@@ -31,6 +31,15 @@ sub run {
     my $namespace = "kiosk";
     my $helm_values = "kiosk_values.yaml";
     my $helm_chart = get_required_var("HELM_CHART");
+
+    my $set_options = "";
+    if (my $image = get_var('CONTAINER_IMAGE_TO_TEST')) {
+        my ($repository, $tag) = split(':', $image, 2);
+        my $helm_values_image_path = get_required_var('HELM_VALUES_IMAGE_PATH');
+
+        $set_options .= "--set $helm_values_image_path.image.repository=$repository --set $helm_values_image_path.image.tag=$tag";
```

Comment:
Since you require both the `reposity` and `tag` to be set, and it might not be clear for users that this is both NEEDED, I would add a check if the `split` function has values for both of them and otherwise fail with a `die`.

Reply:
We reached an agreement in Slack that this will be part of a future PR, soon to be implemented.